vpr/src/route/route_common.cpp

```diff
@@ -698,11 +675,11 @@ void reset_path_costs(const std::vector<int>& visited_rr_nodes) {
 }
 
 /* Returns the *congestion* cost of using this rr_node. */
```

Comment:
Update this comment:
/* Returns the congestion cost of using this rr-node plus that of any non-configurably 
 * connected rr_nodes that *must* be used when it is used.
 */